Bash_scripts/HW20/roles/hw202/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+---
+#Get info from files
+- name: "Get variables from file"
+  import_tasks: template.yml
+
+#Copy to group linux
+- name: "Copy files to hosts"
+  become: true
+  become_user: root
+  ansible.builtin.copy:
+   src: "{{ src_curl }}"
```

Comment:
Current task implemented for single file copy. Improve for multiple file copy. Use `loop`

Reply:
done. I add loop for copy task